paasta_tools/cli/cmds/rollback.py

```diff
@@ -79,54 +82,54 @@ def add_subparser(subparsers):
 def list_previously_deployed_shas(parsed_args, **kwargs):
     service = parsed_args.service
     soa_dir = parsed_args.soa_dir
-    deploy_groups = {deploy_group for deploy_group in parsed_args.deploy_groups.split(',') if deploy_group}
-    return (sha for sha, _ in get_git_shas_for_service(service, deploy_groups, soa_dir))
+    return (sha for sha, _ in get_git_shas_for_service(service, parsed_args.deploy_group, soa_dir))
 
 
-def get_git_shas_for_service(service, deploy_groups, soa_dir):
+def get_git_shas_for_service(service, deploy_group, soa_dir):
     """Returns a dictionary of 2-tuples of the form (timestamp, deploy_group) for each deploy sha"""
     if service is None:
         return []
     git_url = get_git_url(service=service, soa_dir=soa_dir)
-    all_deploy_groups = list_deploy_groups(
-        service=service,
-        soa_dir=soa_dir,
-    )
-    deploy_groups, _ = validate_given_deploy_groups(all_deploy_groups, deploy_groups)
     previously_deployed_shas = {}
     for ref, sha in list_remote_refs(git_url).items():
         regex_match = extract_tags(ref)
         try:
-            deploy_group = regex_match['deploy_group']
+            dg = regex_match['deploy_group']
```

Comment:
Is there a more self-documenting name instead of `dg`?  Something like `ref_deploy_group`? It took me a while to realize what this code was doing.